packages/errors/src/errors.ts

```diff
@@ -0,0 +1,1424 @@
+
+        ${fmt.off(arg1.link)}`
+  },
+  PLAN_EXCEEDS_MONTHLY_TESTS: (arg1: {link: string, planType: string, usedTestsMessage: string, limit: number}) => {
+    return errTemplate`\
+        You've exceeded the limit of test results under your ${fmt.highlight(arg1.planType)} billing plan this month. ${getUsedTestsMessage(arg1.limit, arg1.usedTestsMessage)}
+
+        To continue getting the full benefits of your current plan, please visit your billing to upgrade.
+
+        ${fmt.off(arg1.link)}`
+  },
+  FREE_PLAN_IN_GRACE_PERIOD_PARALLEL_FEATURE: (arg1: {link: string, gracePeriodMessage: string}) => {
+    return errTemplate`\
+        ${fmt.highlightSecondary(`Parallelization`)} is not included under your free plan.
+
+        Your plan is now in a grace period, which means your tests will still run in parallel until ${fmt.highlight(arg1.gracePeriodMessage)}. Please upgrade your plan to continue running your tests in parallel in the future.
+
+        ${fmt.off(arg1.link)}`
+  },
+  PARALLEL_FEATURE_NOT_AVAILABLE_IN_PLAN: (arg1: {link: string}) => {
+    return errTemplate`\
+        ${fmt.highlightSecondary(`Parallelization`)} is not included under your current billing plan.
+
+        To run your tests in parallel, please visit your billing and upgrade to another plan with parallelization.
+
+        ${fmt.off(arg1.link)}`
+  },
+  PLAN_IN_GRACE_PERIOD_RUN_GROUPING_FEATURE_USED: (arg1: {link: string, gracePeriodMessage: string}) => {
+    return errTemplate`\
+        ${fmt.highlightSecondary(`Grouping`)} is not included under your free plan.
+
+        Your plan is now in a grace period, which means your tests will still run with groups until ${fmt.highlight(arg1.gracePeriodMessage)}. Please upgrade your plan to continue running your tests with groups in the future.
+
+        ${fmt.off(arg1.link)}`
+  },
+  RUN_GROUPING_FEATURE_NOT_AVAILABLE_IN_PLAN: (arg1: {link: string}) => {
+    return errTemplate`\
+        ${fmt.highlightSecondary(`Grouping`)} is not included under your current billing plan.
+
+        To run your tests with groups, please visit your billing and upgrade to another plan with grouping.
+
+        ${fmt.off(arg1.link)}`
+  },
+  // TODO: fix
+  FIXTURE_NOT_FOUND: (arg1: string, arg2: string[]) => {
+    return errTemplate`\
+        A fixture file could not be found at any of the following paths:
+
+          ${fmt.listItem(arg1)}
+          ${fmt.listItem(arg1)}.[ext]
+
+        Cypress looked for these file extensions at the provided path:
+
+          ${fmt.listItem(arg2.join(', '))}
+
+        Provide a path to an existing fixture file.`
+  },
+  // TODO: test this
+  AUTH_COULD_NOT_LAUNCH_BROWSER: (loginUrl: string) => {
+    return errTemplate`\
+      Cypress was unable to open your installed browser. To continue logging in, please open this URL in your web browser:
+
+      ${fmt.url(loginUrl)}`
+  },
+  AUTH_BROWSER_LAUNCHED: () => {
+    return errTemplate`Check your browser to continue logging in.`
+  },
```

Comment:
In #20204`AUTH_COULD_NOT_LAUNCH_BROWSER` and `AUTH_BROWSER_LAUNCHED` were removed from `errors.js`. The reasons were:

1. The new formatting on the front end for `AUTH_COULD_NOT_LAUNCH_BROWSER` goes beyond markdown (URL goes into its own box with a copy button now)
1. `AUTH_BROWSER_LAUNCHED` is not an error, it just seems to be here to get a formatted message for successful launch

Since Auth success and error states are only reflected in the Login Modal, they will never be displayed in a place that would use the formatting added here.

So I think these can be removed from this PR (or if there's a reason to keep them, we should revisit the login modal implementation of these).

Reply:
Got it, I did see that had landed recently. We can remove these here, I'll go ahead and do that